raw_data/preprocessed/africa/kenya_english.txt

```diff
@@ -0,0 +1,471 @@
f use of environmental resources; [d] conduct environmental audits; and [e] perform such other functions as may be required under this Act or under the Gazette Notice appointing him. [3] An environmental inspector may, in the performance of his duties under this Act or any regulations made thereunder, at all reasonable times and without a warrant: [a] enter any land, premises, vessel, motor vehicle or ox-drawn trailer and make examinations and enquiries to determine whether the provisions of this Act are being complied with; [b] require the production of, inspect, examine and copy licences, registers, records and other documents relating to this Act or any other law relating to the environment and the management of natural resources; [c] take samples of any articles and substances to which this Act relates and, as may be prescribed, submit such samples for test and analysis; [d] carry out periodic inspections of all establishments and undertakings within their respective jurisdictional limits which manufacture, produce as by-products, import, export, store, sell, distribute or use any substances that are likely to have significant impact on the environment, to ensure that the provisions of this Act are complied with; [e] seize any article, vessel, motor vehicle, plant, equipment, substance or any other thing which he reasonably believes has been used in the commission of an offence under this Act or the regulations made thereunder; [f] with the written approval of the Director-General order the immediate closure of any manufacturing plant or other establishment or undertaking which pollutes or is likely to pollute the environment contrary to the provisions of this Act and to require the owner or operator of such establishment or undertaking to implement any remedial measures that the environmental inspector may direct in the notice closing down the establishment or undertaking. Any establishment or undertaking closed down under this paragraph may resume its operations only with the written approval of the DirectorGeneral; [g] with the approval of the Director-General issue an improvement notice requiring the owner or operator of any manufacturing plant, vessel, motor vehicle or other establishment or undertaking to cease any activities deleterious to the environment and to take appropriate remedial measures, including the installation of new plant and machinery if necessary, within such reasonable time as the DirectorGeneral may determine; [h] with an arrest warrant and the assistance of a police officer, arrest any person whom he reasonably believes has committed an offence under this Act; and [i] install any equipment on any land, premise, vessel or motor vehicle for purposes of monitoring compliance with the provisions of this Act, or the regulations made thereunder upon giving the owner or occupier of the land three months written notice. [4] In exercising his powers under this Act, the environmental inspector shall suitably identify himself.
```

Comment:
Please replace `undertaking. Any establishment` with `undertaking; any establishment`. Also, please replace both instances of `DirectorGeneral` with `Director-General`.